pkg/util/cns.go

```diff
@@ -10,6 +10,19 @@ import (
 	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha5"
 )
 
+const (
+	// IDEControllerMaxSlotCount is the maximum number of slots
```

Comment:
There is a review from https://github.com/vmware-tanzu/vm-operator/pull/1244/files#r2469798582. I will update this as well once the mentioned PR is merged to align with the change.